views/pages/RegisterPage.jsx

```diff
@@ -118,8 +117,6 @@ const RegisterPage = () => {
                 onInputChange={handleChange}
               />
             </div>
-
-            {width < MIN_WIDTH ? null : <CardMedia image="/img/dpdash.png" />}
```

Comment:
do we need a media query here to hide/show the `CardMedia`?

Reply:
This page will change so I would say this is out of scope.